TempleEight/docs/installation.md

```diff
@@ -32,24 +32,42 @@ import TabItem from '@theme/TabItem';
   values={[
     { label: 'MacOS', value: 'macos', },
     { label: 'Linux', value: 'linux', },
-    { label: 'Windows', value: 'windows', },
+    { label: 'Docker Image', value: 'docker', },
   ]
 }>
+
 <TabItem value="macos">
 Installation is via <a href="https://brew.sh/">Homebrew</a>.
 To install, run:
-<code>~ ❯❯❯ brew tap templeeight/temple</code><br/>
-<code>~ ❯❯❯ brew install temple</code>
+<code>❯❯❯ brew tap templeeight/temple</code>
+<code>❯❯❯ brew install temple</code>
+</TabItem>
+
+<TabItem value="linux">
+Install by grabbing the latest release from <a href="https://github.com/TempleEight/temple/releases">GitHub</a>:
+<code>❯❯❯ wget -O /usr/bin/temple https://github.com/TempleEight/temple/releases/download/v0.1.0/temple-latest </code>
+<code>❯❯❯ chmod +x /usr/bin/temple</code>
+</TabItem>
+
+<TabItem value="docker">
+A Docker image is available on Docker Hub:
+<code>❯❯❯ docker run templeeight/temple:0.1 </code>
```

Comment:
Might be worth mentioning that you have to mount the volume into the container to get persistent data